integration-tests/features/steps/stack_analysis.py

```diff
@@ -79,6 +79,19 @@ def wait_for_stack_analysis_completion(context, version=3, token="without"):
     context.duration = end_time - start_time
 
 
+@when("I wait for dynamic stack analysis version {version:d} to finish {token} authorization token")
+def wait_for_stack_analyses_dynamic(context, attempt=1, version=3, token="with"):
+    """Stack Analyses for dynamic Manifests functionality."""
+    try:
+        wait_for_stack_analysis_completion(context, version=version, token=token)
+    except Exception:
```

Comment:
@deepak1725 - I am not aware whether this exception is going to be generated in case of test failures. Else looks good to me

Reply:
@miteshvp Here it is https://ci.centos.org/job/devtools-f8a-master-deploy-e2e-test/2403/consoleFull